pkg/fakerp/server.go

```diff
@@ -101,6 +101,19 @@ func (s *Server) validate(w http.ResponseWriter, r *http.Request) bool {
 }
 
 func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
+
+	config := &api.PluginConfig{
+		AcceptLanguages: []string{"en-us"},
```

Comment:
Not necessary now but in the future we can keep this inside a `map[resourceGroup]PluginConfig; type resourceGroup string` so that it can stay in sync between PUT and DELETE.